R/react.R

```diff
@@ -0,0 +1,103 @@
+#' Create a React element
+#'
```

Comment:
1 sentence description needed - when to use this

Reply:
Done.